Source/VSProps/QtCompile.props

```diff
@@ -32,7 +33,7 @@
       -->
       <AdditionalOptions>%(AdditionalOptions) /experimental:external</AdditionalOptions>
       <AdditionalOptions>%(AdditionalOptions) /external:W0</AdditionalOptions>
-      <AdditionalOptions>%(AdditionalOptions) /external:I$(QtIncludeDir)</AdditionalOptions>
+      <AdditionalOptions>%(AdditionalOptions) /external:I "$(QtIncludeDirWithoutTrailingSeparator)"</AdditionalOptions>
```

Comment:
If I put quotes around QtIncludeDir the trailing separator escapes the second quote and causes VS to barf in new and exciting ways.  I could put \" afterward so the trailing separator escapes the \ instead, which ends up with the right result, but I think this way is clearer (if more verbose).